<launch>
    <rosparam file="$(find iToF_base)/config/param/param_li.yaml" command="load"/>

    <param name="save_dir_up" type="string" value="$(find iToF_base)/result/li/li_iToF_up.yaml" />
    <param name="save_dir_right" type="string" value="$(find iToF_base)/result/li/li_iToF_right.yaml" />
    <param name="save_dir_left" type="string" value="$(find iToF_base)/result/li/li_iToF_left.yaml" />

    <!-- ground_reference??? -->
    <param name="is_ground_reference" type="bool" value="false" />

    <node pkg="iToF_base" type="iToF_base" name="iToF_base" output="screen"/>

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find iToF_base)/config/rviz/li.rviz"/>

</launch>